src/modelbench/hazards.py

```diff
@@ -225,5 +236,19 @@ def reference_standard_for(self, name):
             raise ValueError(f"No standard yet for {name}. Run `modelbench calibrate --update` to add one.")
         return self.data["reference_standards"][name]
 
+    def average_standard_across_references(self, locale: str = "") -> float:
+        values = []
+        if locale:
+            # refs with locale are all version 1.0 or newer
+            locale = locale.lower()
+            values = [v for k, v in self.data["reference_standards"].items() if locale in k]
+        else:
+            # no locale means we want 0.5
```

Comment:
I don't love that this is implicit, can we add a version arg?

Reply:
Good point! Added.